test/test.tf

```diff
@@ -2,7 +2,7 @@ locals {
   api_domain  = format("gw%s%s", var.envid, var.domain_sfx)
   api_aliases = [format("alias%s%s", var.envid, var.domain_sfx)]
   lb_fqdn     = format("lb%s%s", var.envid, var.domain_sfx)
-  mdw_fqdn    = "testnet.aeternal.io"
+  mdw_fqdn    = "testnet.aeternity.io"
```

Comment:
this is wrong, there is no running /middleware behind this domain, it should be `mdw.testnet.aeternity.io` as well IMO

Reply:
FYI: it works ATM as the gateway is still running the old configuration